### PR TITLE
Add libdiskconfig_host_grub.a build rule

### PR DESCRIPTION
**Pull request by a-k-v**
JIRA: AIA-341
Tests: Check if install_mbr is built in out/host
by building from bootable/liveinstaller & grub binary
boots & installs fine.

Signed-off-by: Abhilash K V <abhilash.k.v@intel.com>

### CODE REVIEW

File: Android.mk [hunk 1]
```diff
@@ -1,10 +1,24 @@
 LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 
+ifeq ($(HOST_OS),linux)
+include $(CLEAR_VARS)
+LOCAL_SRC_FILES := \
+		../../system/core/libdiskconfig/diskconfig.c \
```
Comment (kalyankondapally) [inline Android.mk]: Instead of linking to the files in system/core, can we just add initial version of diskconfig(forking from system/core) here.

Comment (a-k-v) [inline Android.mk]: OK , in that case how do we ensure that future changes to the source files would get reflected here ? libdiskconfig_host & libdiskconfig libs in system/core are same (host vs target libs).  If we fork then libdiskconfig_host_grub would differ from libdiskconfig.

Comment (kalyankondapally) [inline Android.mk]: hmm ya, was thinking if we can just stick to existing version and not change it till something is really needed. But what you said makes sense. We can go with this for now I guess

**Comment by kalyankondapally** [comment]
@Kishore409 Can u merge this ?

**Comment by Kishore409** [comment]
@kalyankondapally @a-k-v we have verified this and install_mbr

**Comment by Kishore409** [comment]
Merged, Thanks

**Pull request merged**